zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -605,9 +605,10 @@ where
     ParamsT: consensus::Parameters + Clone,
     FeeRuleT: FeeRule,
 {
+    unimplemented!();
     let mut step_results = Vec::with_capacity(proposal.steps().len());
     for step in proposal.steps() {
-        let step_result = create_proposed_transaction(
+        let step_result = calculate_proposed_transaction(
```

Comment:
Should the caller `create_proposed_transactions` be updated `calculate_proposed_transactions`